src/workers/event.rs

```diff
@@ -1427,11 +1430,17 @@ mod test {
                                     annotations: None,
                                     instance_id: "1".to_string(),
                                     revision: 0,
+                                    // TODO: FIXME
+                                    image_ref: None,
+                                    max_concurrent: 1,
```

Comment:
For all of these, leaving the image_ref as `None` and max_concurrent as 0 simulates backwards-compatible behavior. I think we should do that until we officially update to v0.79.0 and use these fields for realsies